tests/unit/rankers/test_algorithms.py

```diff
@@ -256,9 +256,9 @@ def test_invalid_trailing_period(self):
 
     @pytest.mark.parametrize("thompson_sampling_function,metrics", [
         (thompson_sampling, generate_metrics(28)),  # 28 day is the default
-        (thompson_sampling_1day, generate_metrics(1)),
-        (thompson_sampling_7day, generate_metrics(7)),
-        (thompson_sampling_14day, generate_metrics(14)),
+        # (thompson_sampling_1day, generate_metrics(1)),
+        # (thompson_sampling_7day, generate_metrics(7)),
+        # (thompson_sampling_14day, generate_metrics(14)),
```

Comment:
I forgot to remove some debug changes:
```suggestion
        (thompson_sampling_1day, generate_metrics(1)),
        (thompson_sampling_7day, generate_metrics(7)),
        (thompson_sampling_14day, generate_metrics(14)),
```